### DISCUSSION TITLE
Pxe boot is bad!

### DISCUSSION THREAD

**Discussion by orlovskyjavaprofi** [ipxe/ipxe#1344] [General]
I have dell server, and you know what it sux to disable your pxe boot.
It was necessary to boot my Linux server, somehow Mellanox loaded the ipxe and it stucked, this problem appears on Dell hardware only.

**Comment by NiKiZe** [comment]
Upgrade to working firmware

**Reply by orlovskyjavaprofi** [reply]
> Upgrade to working firmware

Yeah and that sux, I will remove the iPXE boot from the network card, i will recommend to do that any my customers whatever they run in theirs infrastructure, **iPXE is obsolete.**

**Reply by NiKiZe** [reply]
The great thing with open source is that you can do what you want. But if you don't want any PXE boot code you can usually disable option ROMs in firmware or BIOS.

I don't know what scenario you have, but iPXE is far from obsolete.

Please direct your anger of your hardware not behaving to those responsible, in this case Dell and maybe, but probably not, Mellanox.

Do you usually respond like this for other software components as well? (Or have you never had issues with the Linux kernel?)

**Comment by mcb30** [comment]
@orlovskyjavaprofi Please try to behave like an adult.
